ytxp-plutarch/src/Cardano/YTxP.hs

```diff
@@ -40,6 +45,13 @@ data YTxPParams = YTxPParams
   deriving stock (Show, Generic, Eq)
   deriving anyclass (ToJSON, FromJSON)
 
+
```

Comment:
Can you add comment why we use this instead of `Ply.#`?

Reply:
Added